src/mlpack/methods/ann/layer/flexible_relu_impl.hpp

```diff
@@ -103,7 +103,10 @@ void FlexibleReLUType<MatType>::Forward(
 
 template<typename MatType>
 void FlexibleReLUType<MatType>::Backward(
-    const MatType& input, const MatType& gy, MatType& g)
+    const MatType& input,
+    const MatType& /* output */,
+    const MatType& gy,
+    MatType& g)
 {
   // Compute the first derivative of FlexibleReLU function.
   g = gy % arma::clamp(arma::sign(input), 0.0, 1.0);
```

Comment:
Could the same thing have been done with input? e.g.
`g = gy % (input > 0)`
It seems better to use input here, since we want the first derivative of the function that is based on the input, not the output.  If alpha were to be negative (not sure if that is possible), this would produce incorrect results.

Reply:
Oh, actually, yeah, I stumbled there, `input` is the right one to use.  I wonder why I wrote `output` instead?